src/App.js

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
+import { Route } from 'react-router-dom';
 import './App.css';
 import { Home, About, Contact, Navigation } from './components';
 
+
 const App = () => (
   <div>
     <Navigation />
+    <Route exact path="/" component={Home}></Route>
+    <Route exact path="/about" component={About}></Route>
+    <Route exact path="/contact" component={Contact}></Route>
+
```

Comment:
Well done on the Routing! I would keep the About and Contact components as just 'path' rather than 'exact path', and keep the only exact path as the home component